PWGJE/Tasks/nucleiInJets.cxx

```diff
@@ -170,14 +170,14 @@ struct nucleiInJets {
   Configurable<bool> applySkim{"applySkim", false, "Apply skimming"};
   Configurable<std::string> cfgSkim{"cfgSkim", "fHighFt0Mult", "Configurable for skimming"};
 
-  static constexpr float gMassProton = 0.93827208f;
-  static constexpr float gMassDeuteron = 1.87561f;
-  static constexpr float gMassTriton = 2.80892f;
-  static constexpr float gMassHelium = 2.80839f;
+  static constexpr float kMassProton = o2::constants::physics::MassProton;
+  static constexpr float kMassDeuteron = o2::constants::physics::MassDeuteron;
+  static constexpr float kMassTriton = o2::constants::physics::MassTriton;
+  static constexpr float kMassHelium = o2::constants::physics::MassHelium3;
```

Comment:
Please avoid defining new unnecessary constants and use the existing ones directly.